archive/100-theorems-list/16_abel_ruffini.lean

```diff
@@ -0,0 +1,258 @@
+/-
+Copyright (c) 2021 Floris van Doorn. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Thomas Browning
+-/
+import field_theory.abel_ruffini
+import analysis.calculus.local_extr
+/-!
+Construction of an algebraic number that is not solvable by radicals.
```

Comment:
Could you put here a sketch of the construction you are using, with the main steps of the argument and references to the Lean names below?